apkid/rules/apk/packers.yara

```diff
@@ -604,3 +604,23 @@ rule secenh : packer
     and 1 of ($a*)
     and 1 of ($b*)
 }
+
+// There can also be rules for dex files because of NRV compression but I can't find compressed classesN.dex
```

Comment:
Is this comment needed in the source code? What about to leave it out from there and write here in the issue?